charts/microgateway/README.md.gotmpl

```diff
@@ -226,431 +231,372 @@ In case that the [Simple DSL configuration](#simple-dsl-configuration) does not
-A route object must be created for its configuration. The creation of such an object is handled by the Microgateway Helm chart.  
-
-#### Without TLS
-This example shows how to configure a route object without TLS using the Helm chart.
-
-```
-route:
-  enabled: true
-  hosts:
-    - virtinc.com
-  targetPort: http
-  tls:
-    enabled: false
-```
-
-#### TLS Configuration
-There are three different TLS configuration options for the openshift route. These are edge, reencrypt and passthrough.  
-Below is a description of how these options can be configured using the Microgateway chart.  
-
-Evtl. nur eine Möglichkeit. 
-There are two different options for specifying TLS certificates in the Helm chart.   
-Either you create a secret (config.tlsSecretName) containing the required certificates or you provide the certificates in a custom-values.yaml file during deployment.   
-
-##### Edge
-With the edge termination TLS is terminated on the router.   
-The TLS certificates are served by the router and must therefore be configured on the router.   
-
-```
-route:
-  enabled: true
-  hosts:
-    - virtinc.com
-  tls:
+The examples are based on the standard Openshift Route. Since this is already available in a Openshift environment, nothing special needs to be done. The Microgateway Helm chart allows to create a Route object.
+
+#### Route terminating HTTP
+
+  To receive HTTP traffic from the outside of the Openshift cluster, use the following configuration:
+  ```
+  route:
     enabled: true
-    termination: edge
-    certificate: |
-      -----BEGIN CERTIFICATE-----
-      MIIDizCCAnOgAwIBAgIJAMQE1QewYs4QMA0GCSqGSIb3DQEBCwUAMFwxCzAJBgNV
-      BAYTAkNIMQ8wDQYDVQQIDAZadXJpY2gxDzANBgNVBAcMBlp1cmljaDEQMA4GA1UE
-      [...]
-      CgwHQWlybG9jazEZMBcGA1UEAwwQdGVzdC5jZXJ0aWZpY2F0ZTAeFw0xNjAyMTYx
-      77RRptcoQJPvw50z9rJ4wkrb58raUKOqxgvpckQdYdtok0dR6tXbBfC4LHmqq0mo
-      -----END CERTIFICATE-----
-    key: |
-      -----BEGIN RSA PRIVATE KEY-----
-      MIIEpAIBAAKCAQEAtXkTjHtDtutxyo1R6N4Eh18IzxoagAHPRzsdB5yeadcVr/bV
-      oKV9XXmvx/VFEPOxyG8q4y/zU0qaESIF8FzOfdPciULVyMY2fWrrEByyQSpfhDvn
-      [...]
-      pN5G7+/iMFWMp3sGaheIGpVJfMV0Mq3t6rOPrjcvNx9tY5dtmno+DUxyPI1jEc1y
-      pGKu7aodwB4cD5YnfXTvUcTv5tNU0llRLG1J0bg1n9cCo0nTC9sUZw==
-      -----END RSA PRIVATE KEY-----       
-```
-
-##### Reencrypt
-Reencrypt is similar to edge termination, because TLS termination takes place on the router.  
-However, with reencrypt the connection is encrypted again before the request is sent to the Microgateway service.   
-This means that the entire path of the connection is encrypted, including the internal network.   
-
-```
-route:
-  enabled: true
-  hosts:
-    - virtinc.com
-  tls:
+    hosts:
+      - virtinc.com
+    targetPort: http
+    tls:
+      enabled: false
+  ```
+
+#### Route terminating secure HTTPS
+Openshift has there three different TLS termination types which can be used. Edge, reencrypt and passthrough.
```

Comment:
Openshift has there three --> Openshift has three ...
or 
In Openshift there are three ...

Reply:
Done